dz_s2/zadacha_1_dz.py

```diff
@@ -0,0 +1,25 @@
+# Напишите программу, которая получает целое число и возвращает его шестнадцатеричное строковое представление.
+# Функцию hex используйте для проверки своего результата.
+
+
+dev = 16
+dict_hex = {10: 'A', 11: 'B', 12: 'C', 13: 'D', 14: 'E', 15: 'F'}
```

Comment:
Можно добавить остальные ключи-цифры из этой системы или вообще использовать строку. На код это практически не повлияет
```suggestion
dict_hex = '0123456789abcdef'
```